devrelservices-admin/main.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	drghs_v1 "github.com/GoogleCloudPlatform/devrel-services/drghs/v1"
+
+	"cloud.google.com/go/errorreporting"
+	"cloud.google.com/go/profiler"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+// Flags
+var (
+	listen  = flag.String("listen", ":6343", "listen address")
+	verbose = flag.Bool("verbose", false, "enable verbose debug output")
+	smpSpr  = flag.String("smp-spr", "samplrd-sprvsr", "address of samplrd-sprvsr")
+	mtrSpr  = flag.String("mtr-spr", "maintnerd-sprvsr", "address of maintnerd-sprvsr")
+)
+
+var (
+	errorClient *errorreporting.Client
+)
+
+// Log
+var log *logrus.Logger
+
+func init() {
+	log = logrus.New()
+	log.Formatter = &logrus.JSONFormatter{
+		FieldMap: logrus.FieldMap{
+			logrus.FieldKeyTime:  "timestamp",
+			logrus.FieldKeyLevel: "severity",
+			logrus.FieldKeyMsg:   "message",
+		},
+		TimestampFormat: time.RFC3339Nano,
+	}
+
+	log.Out = os.Stdout
+}
+
+func main() {
+	flag.Parse()
+
+	if *verbose {
+		log.Level = logrus.TraceLevel
+	}
+
+	if *listen == "" {
+		log.Fatal("error: must specify --listen")
+	}
+
+	if *smpSpr == "" {
+		log.Fatal("error: must specify --smp-spr")
+	}
+
+	if *mtrSpr == "" {
+		log.Fatal("error: must specify --mnt-spr")
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	signalCh := make(chan os.Signal, 1)
+	signal.Notify(signalCh, os.Interrupt)
+	go func() {
+		sig := <-signalCh
+		log.Printf("termination signal received: %s", sig)
+		cancel()
+	}()
+
+	if err := profiler.Start(profiler.Config{
+		Service:        "devrel-admin",
+		ServiceVersion: "0.0.1",
```

Comment:
`0.0.1` => `v1` yes?

Reply:
That version is an arbitrary string and doesn't correlate with anything in particular (it can be empty string). 

Also, the profiler doesn't map to Endpoints or other services. Thinking about this more, I might consider leaving it as empty string just because in production we will not have multiple versions of the application running concurrently.